src/classlibnative/bcltype/bignum.cpp

```diff
@@ -0,0 +1,608 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// File: bignum.cpp
+//
+
+//
+
+#include "bignum.h"
+
+#if !defined(FEATURE_PAL)
+#include <intrin.h>
```

Comment:
Please make this include unconditional. We have an `intrin.h` for `FEATURE_PA`L which ends up including `pal.h`.